plugins/source/scheduler_dfs.go

```diff
@@ -161,6 +162,14 @@ func (p *Plugin) resolveResourcesDfs(ctx context.Context, table *schema.Table, c
 				if err := resolvedResource.Validate(); err != nil {
 					tableMetrics := p.metrics.TableClient[table.Name][client.ID()]
 					p.logger.Error().Err(err).Str("table", table.Name).Str("client", client.ID()).Msg("resource resolver finished with validation error")
+					if _, found := sentValidationErrors.LoadOrStore(table.Name, struct{}{}); !found {
```

Comment:
do we we want only to send once per table? I guess it makes sense even though we can miss some things but yeah let's start with that.

Reply:
Yeah, we may miss some things, but I hope not. And we'll catch it once the first issue is fixed anyway. I figure it's better than using our entire Sentry quota in one sync (unless I'm misunderstanding how Sentry billing works)